src/libraries/System.Security.Cryptography/tests/X509Certificates/PfxIterationCountTests.CustomAppDomainDataLimit.cs

```diff
@@ -0,0 +1,140 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.RemoteExecutor;
+using Microsoft.DotNet.XUnitExtensions;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    [SkipOnPlatform(TestPlatforms.Browser, "Browser doesn't support X.509 certificates")]
+    // AppContext and AppDomain are the same in this context.
+    public class PfxIterationCountTests_CustomAppDomainDataLimit
```

Comment:
.NET Core/.NET 5+ doesn't have AppDomains, presumably the class, file, and all the method names want to replace "AppDomain" with "AppContext"?